numpy/lib/info.py

```diff
@@ -147,6 +147,8 @@
 setxor1d         Set exclusive-or of 1D arrays with unique elements.
 in1d             Test whether elements in a 1D array are also present in
                  another array.
+isin             Test whether elements in an array are also present in
+                 another array, preserving shape of first array.
```

Comment:
Maybe "test whether each element of one array is present anywhere within another"?

No strong feelings here